### PR TITLE
Final changes for a "clean" Section 3.

### LINKED ISSUE #738: Experiment: get current WHATWG/W3C Section 3
Using https://diffofhtmls.herokuapp.com/ Get Section 3 current (or explain the differences).

Following this activity, we should know:
* How long the experiment took (in hours)?
  * **Results: 28.7 hours**
* What are the remaining differences (if any)? 
  * Issue #748 - Continue to define xml:base as [not] for HTML docs? 
  * Issue #751 - Accept WHATWG change to drop restrictions on the Bidi algorithm? 
  * Known Difference: We have a detailed section on ARIA roles, states, and properties (mostly non-normative) 
  * Known Difference: We don't define `accessKeyLabel` of the `HTMLElement` interface
* Why there are remaining differences?

### LINKED ISSUE #739: Make the spec consistent "standard" vs. "specification"
There are places where the spec describes a XXX standard and other places where it is referred to as XXX specification.

Tentative agreement is to unify on the term "specification"

### PR DESCRIPTION
**Pull request by travisleithead**
Fix #738
Fix #739
Much linking improvement in Section 3 (and beyond).

### CODE REVIEW

File: sections/infrastructure.include [hunk 1]
```diff
@@ -241,17 +241,17 @@
 
   A <dfn>character encoding</dfn>, or just
   <i>encoding</i> where that is not ambiguous, is a defined way to convert between byte streams and
-  Unicode strings, as defined in the WHATWG Encoding standard. An <a>encoding</a> has an
+  Unicode strings, as defined in the WHATWG Encoding specification. An <a>encoding</a> has an
```
Comment (arronei) [inline sections/infrastructure.include:244]: WHATWG doesn't have specifications it has standards. I think you will want to change this text

File: sections/infrastructure.include [hunk 2]
```diff
@@ -241,17 +241,17 @@
...
+  encoding's <i>name</i> and <i>labels</i> in the Encoding specification. [[!ENCODING]]
 
   A <dfn>UTF-16 encoding</dfn> is <a>UTF-16BE</a> or <a>UTF-16LE</a>. [[!ENCODING]]
 
   An <dfn>ASCII-compatible encoding</dfn> is any <a>encoding</a> that is not a
   <a>UTF-16 encoding</a>. [[!ENCODING]]
 
   <p class="note">
-    Since support for encodings that are not defined in the WHATWG Encoding standard is prohibited,
+    Since support for encodings that are not defined in the WHATWG Encoding specification is prohibited,
```
Comment (arronei) [inline sections/infrastructure.include:254]: Multiple cases of the "standard" issue

File: sections/semantics-forms.include [hunk 3]
```diff
@@ -538,7 +538,7 @@
   user's locale, to allow scripts in pages and on servers to process times, dates, and
   numbers in a consistent manner with minimal work.
 
-  In HTML markup and form submission dates and times are always written in a locale-neutral format derived from the ISO-8601 standard. For example, dates are generally in a format such as 2016-06-19.
+  In HTML markup and form submission dates and times are always written in a locale-neutral format derived from the ISO-8601 specification. For example, dates are generally in a format such as 2016-06-19.
```
Comment (arronei) [inline sections/semantics-forms.include]: Line wrapping

**Review by arronei — CHANGES_REQUESTED** [review]

File: sections/infrastructure.include (hunk 1 above)
Comment (travisleithead) [reply sections/infrastructure.include:244]: Fixed in more recent commit

File: sections/infrastructure.include (hunk 2 above)
Comment (travisleithead) [reply sections/infrastructure.include:254]: Ditto above.

**Comment by travisleithead** [comment]
Comments all addressed. Local-building before committing...

**Pull request merged**